challenges/classes.js

```diff
@@ -1,7 +1,50 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+	constructor(length, width, height) {
+		this.length = length;
+		this.width = width;
+		this.height = height;
+	}
+
+	volume() {
+		return `${this.length}` * `${this.width}` * `${this.height}`;
+	}
+
+	surfaceArea() {
+		return (
+			2 *
+			(`${this.length}` * `${this.width}` +
+				`${this.length}` * `${this.height}` +
+				`${this.width}` * `${this.height}`)
+		);
+	}
+}
+
 // Test your volume and surfaceArea methods by uncommenting the logs below:
 // console.log(cuboid.volume()); // 100
 // console.log(cuboid.surfaceArea()); // 130
 
-// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
\ No newline at end of file
+// Stretch Task: Extend the base class CuboidMaker with a sub class called CubeMaker.  Find out the formulas for volume and surface area for cubes and create those methods using the dimension properties from CuboidMaker.  Test your work by logging out your volume and surface area.
+
+class CubeMaker extends CuboidMaker {
+	constructor(length, width, height) {
+		super(length, width, height);
+		this.length = length;
+		this.width = width;
+		this.heigth = height;
+	}
+
+	cubeVolume() {
+		return 3 * `${this.length}` * `${this.width}` * `${this.height}`;
+	}
+
+	cubeSurfaceArea() {
+		return (
+			6 *
+			(`${this.length}` * `${this.width}` +
+				`${this.length}` * `${this.height}` +
+				`${this.width}` * `${this.height}`)
```

Comment:
Nice job on the stretch, a couple of things here prevent this from fully working and passing the tests, however a solid effort. 15/25